itests/fe/permissions_test.py

```diff
@@ -62,7 +62,7 @@ def create_test_data(setup):
 def test_list(tmpdir, setup, browser):
     # type: (LocalPath, SetupTest, Chrome) -> None
     permissions = create_test_data(setup)
-    template_engine = FrontendTemplateEngine(FrontendSettings(), "")
+    template_engine = BaseTemplateEngine(Settings(), "grouper.fe")
```

Comment:
Since this is a test of the frontend's output, I think this should still be a `FrontendTemplateEngine`. In case, for example, we want to use different `print_date` functions between the fe and emails

Reply:
The problem is that `FrontendTemplateEngine` has various other prerequisites that I didn't want to initialize, since this test isn't really using the template engine at all, just the date formatting. Instead, I'll just copy the single line of date formatting logic over to the test, since it's highly unlikely to change, and avoid instantiating any template engine at all.